### ISSUE TITLE
Can it be possible to have multiple source folders?

### ISSUE THREAD

**Issue by LadyHaywood** [komikku-app/komikku#182]
### Describe your suggested feature

This is something I've only known J2K to do even before the rework of how source folders functioned. This feature(?) really important and helpful for me because of the device I use. Sorry if this is confusing, but my e-reader will show all supported book files on the library's home screen which is almost what I want but that means it'll show _**all**_ of the chapters and it doesn't let you exclude file types or remove them from the library in bulk.

My workaround for this has been to put every book's "chapter 1" in the J2K's local folder and then put the rest of the chapters in the Tachiyomi's local folder since J2K can read both folders and just merges them together in the J2K app. That way my library's home screen isn't cluttered with "[name of manga], ch.2, ch.3" and so on. It only shows the 1st chapter with it's metadata. That's also why there isn't a manga title for ch.1 in the image for J2K because it's an epub.

Again, sorry if this sounds weird. If it helps, this is basically what I mean. Dear Gene's chapter 1 is the only thing showing in the library while the other chapters aren't since they're in a different folder but J2K shows all chapters together because it can read both folders.
Is this something that can be done with the new file system in the future?
![865482](https://github.com/user-attachments/assets/55aeab7b-bddf-4087-b43a-2f08b108825c)
![87534](https://github.com/user-attachments/assets/57f871e2-bf59-47cc-8718-64f006fc97f0)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an official extension, I should be opening an issue in the [extensions repository](https://github.com/komikku-app/extensions/issues/new/choose).
- [X] I have updated the app to version **[1.10.5](https://github.com/komikku-app/komikku/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by cuong-tran** [comment]
Could you chat in Discord so I can understand your issue?

**Comment by LadyHaywood** [comment]
I don't really have a discord account anymore. I had closed it, I'm sorry. I'll admit that was more of a selfish ask of me because of the device I'm using and I was a bit wordy about it.

What I was trying to say was: J2K has the ability to read local files from their "TachiyomiJ2K" folder and the original "Tachiyomi" folder. I was asking for something so it would be easier to make my ereader's home page look cleaner when I had moved over from using J2K.

Again, a selfish ask that doesn't add better functionality and organization to the app itself. It was just a J2K feature(?) that I utilized because make organizing on the ereader easier. It would be nice to have but, honestly, it's not that important.

**Comment by cuong-tran** [comment]
Let me put it like this, correct me if i'm wrong.
- You e-reader home screen is showing all files which you put in Tachiyomi/local folder
- To avoid it shows all chapters separately, you move the 2nd & other chapters to J2K/local folder
- By doing so, you e-reader home screen now only show the first chapter
- Mean while, J2K will show files from both Tachiyomi/local and J2K/local & merge them together under 1 manga entry

**Comment by LadyHaywood** [comment]
Yeah, exactly that but the other way around. Chapter 1, detail file, and cover would go into J2K's local folder since I didn't have to worry about images and json files showing up on the home page. At least not on boox devices anyway.

But yeah, that was how I organized it since ereaders don't let you exclude book file types. Devices like Boox let you remove them once they're there but not in a very easy way and I have over 500 manga files..

**Comment by cuong-tran** [comment]
Does your ereader show up files if they are images, like jpg or png?
If not, try turn off saving as CBZ before download chapters.

**Comment by cuong-tran** [comment]
Then try turn off save as CBZ in Download section

**Comment by LadyHaywood** [comment]
I've understand what you're saying and I did that for awhile when setting up local entries, but it took up more space than my current method. I don't directly download from the app itself. I spend the time preparing the entry's details, epub, and cbz files on my computer before moving them over. For the cbz part I try to make the file small without losing quality and remove extra images that aren't the manga pages.

I get that it's more set up work than just downloading them from the app but how I prepare the local manga is preferred. It's kinda why I said, looking back now, I get that this was more of a selfish ask lol.
